app/api/domains/api.py

```diff
@@ -108,6 +109,35 @@ def format_map_basic(m: Beatmap) -> dict[str, object]:
     }
 
 
+@router.get("/search_players")
+async def api_search_players(
+    search: Optional[str] = Query(None, alias="q", min=2, max=32),
+    db_conn: databases.core.Connection = Depends(acquire_db_conn),
+):
+    """Search for users on the server by name."""
+    rows = await db_conn.fetch_all(
+        "SELECT id, name FROM users WHERE name LIKE :search AND priv & 3 = 3 ORDER BY id ASC",
+        {"search": f"%{search}%" if search is not None else None},
+    )
+
+    if len(rows) == 0:
+        return ORJSONResponse(
+            {
+                "status": "error",
+                "message": "nothing was found matching those parameters!",
+            },
```

Comment:
hmm, seems this is inconsistent with my old style - scrolling down i see some like
```py
return ORJSONResponse(
    {"status": "Invalid gamemode."},
    status_code=status.HTTP_400_BAD_REQUEST,
)
```
i actually prefer what you have lol, it's what i use in my newer projects

quite unfortunate

Reply:
pre-commit changed that, originally it was in line with what you have already

Reply:
actually, on second thought, let's return 200 with [] result